pymc4/distributions/state_functions.py

```diff
@@ -0,0 +1,161 @@
+import tensorflow as tf
+import tensorflow_probability as tfp
+from tensorflow_probability.python.mcmc.internal import util as mcmc_util
+
+tfd = tfp.distributions
+
+__all__ = ["categorical_uniform_fn", "bernoulli_fn", "gaussian_round_fn", "poisson_fn"]
```

Comment:
You should remove the `poisson_fn`, as the proposal density is not symmetric which makes working with it more difficult. As implemented right now it is not valid.

Reply:
Yeah, I've commented that above. Ok, I will just remove this proposal for now. Poisson proposal is implemented in pymc3 but not implemented as I understand.